### ISSUE TITLE
Pretrained NLI models

### ISSUE THREAD

**Issue by dirkweissenborn** [uclmr/jack#350]
We need pre-trained models for MultiNLI and SNLI.

- [x]  ESIM SNLI
- [x]  ESIM MultiNLI
- [x]  DAM SNLI
- [x]  DAM MultiNLI
 
... and create some documentation for this task like for [extractive QA](https://github.com/uclmr/jack/blob/master/docs/Extractive_QA.md)

**Comment by pminervini** [comment]
I've tried it once but nothing was working (the very same code is getting ~SOTA - ~87% test accuracy - in another codebase) - let me debug this

**Comment by dirkweissenborn** [comment]
when did you try? the code has changed, maybe it will work better now.

**Comment by pminervini** [comment]
Let's see!

**Comment by pminervini** [comment]
The code is now very very different from what I have - I have to find some time for looking into this new implementation of ESIM (and DAM eventually), and think what it means in terms of time and papers

**Comment by pminervini** [comment]
I'm not sure I've the time of working on two very distinct implementations of the same thing

**Comment by dirkweissenborn** [comment]
It is ok to simply run the current implementations as is, host the trained models and report the results in a specific NLI document. No need to look into the code at the moment. We just need some models and numbers.

**Comment by pminervini** [comment]
ESIM seems decent (86.6, 86.0); DAM seems sub-par (48.0, 48.4), probably it's just a matter of hyperparams.

I'm uploading the models here: http://jack.neuralnoise.com/jack/natural_language_inference/

**Comment by pminervini** [comment]
Done. Happy Holidays!
